feed/index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Feed - The Inn</title>
+    <link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.min.css" />
+    <script src="https://kit.fontawesome.com/57518a1e7d.js" crossorigin="anonymous" defer></script>
+    <script src="../node_modules/bootstrap/dist/js/bootstrap.min.js" defer></script>
+    <script src="../node_modules/bootstrap/dist/js/bootstrap.bundle.min.js" defer></script>
+    <link rel="stylesheet" href="../dist/css/styles.css" />
+  </head>
+  <body>
+    <header class="bg-primary p-2 mb-3">
+      <div class="container p-0">
+        <div class="d-flex row-flex justify-content-between align-items-center">
```

Comment:
In Bootstrap, a navbar should be wrapped with a navbar class. See example at https://getbootstrap.com/docs/5.3/components/navbar/#how-it-works

Reply:
For your info, this is part of the requirements. 

```The submission uses the .navbar component to represent navigation menus.```